service/hook/github/github_test.go

```diff
@@ -489,6 +505,8 @@ func Test_transformPullRequestEvent(t *testing.T) {
 					BranchDest:               "master",
 					PullRequestID:            pointers.NewIntPtr(12),
 					PullRequestRepositoryURL: "https://github.com/bitrise-io/bitrise-webhooks.git",
+					BaseRepositoryURL:        "https://github.com/bitrise-io/bitrise-webhooks.git",
+					HeadRepositoryURL:        "https://github.com/bitrise-io/bitrise-webhooks.git",
```

Comment:
Should we introduce a test where these actually differ? Wdyt?

Reply:
I think we should already be covered on that [here](https://github.com/bitrise-io/bitrise-webhooks/pull/104/files/0176f62b1a8e79f43ae2cfc95df0e3db97430252#diff-38bbaeb8b44fcfd19b95af3b8319e5051d3a40b19d739f5b4fbc722dc416cc66R1009).

Reply:
Ahh, you're right, any chance you can change it to something else, so it's easier to spot the difference? I read the whole diff, I promise :)